src/test/test_utils.hpp

```diff
@@ -283,6 +283,7 @@ static std::vector<T> asVector(const std::string& tensor_content) {
 
 class ConstructorEnabledModelManager : public ovms::ModelManager {
     ovms::MetricRegistry registry;
+    ovms::MetricConfig metricConfig;
```

Comment:
This member shadows ovms::ModelManager::metricConfig?
Does it compile? If so, how can we be sure which one is used in reporter in unit tests?

Reply:
Removed.